lib/worker/src/rpc.rs

```diff
@@ -242,24 +255,26 @@ impl<E: EVMClient + 'static> publications::Server for Publications<E> {
         if rel.is_empty() {
             return Promise::err(Error::failed("relation is required".into()));
         }
-        let n = args.get_n();
+        let n = args.get_n() as i32;
 
-        let e = self.evm_client.clone();
+        let pg_pool = self.pg_pool.clone();
+        let web3storage_client = self.web3storage_client.clone();
         Promise::from_future(async move {
-            let deals = e
-                .latest_deals(format!("{}.{}", ns, rel).as_str(), n)
-                .await?;
+            let cids = db::pub_cids(&pg_pool, ns, rel, n, 0).await?;
```

Comment:
This rpc method turned out to be the same as the above, it is used to return the latest `n` deals. The only difference is what is passed as `limit` and `offset`. 

 Maybe we can remove it later